extensions/Khronos/KHR_materials_pbrSpecularGlossiness/README.md

```diff
@@ -0,0 +1,214 @@
+
+The following example shows how the same material can be defined using both metallic-roughness as well as specular-glossiness material models:
+
+```
+{
+    "materials": [
+        {
+            "name": "gold",
+            "pbrMetallicRoughness": {
+                "baseColorfactor": [ 1.000, 0.766, 0.336, 1.0 ],
+                "metallicFactor": 1.0,
+                "roughnessFactor": 0.0
+            },
+            "extensions": {
+                "KHR_materials_pbrSpecularGlossiness": {
+                    "diffuseFactor": [ 0.5, 0.5, 0.5, 1 ],
+                    "specularFactor": [ 0.0, 0.0, 0.0 ],
+                    "glossinessFactor": 0.8
+                }
+            }
+        }
+    ]
+}
+```
+If the pbrSpecularGlossiness extension is included in an asset, then any runtime implementation that supports the extension should always render the asset using the specular-glossiness material properties to ensure quality.
+
+A glTF asset may also choose to use the pbrSpecularGlossiness extension without including any metallic-roughness properties for its material definition. If such an asset chooses to include the extension using the `extensionUsed` property then a runtime that doesn't support the pbrSpecularGlossiness extension may choose to render the asset by converting the specular-glossiness values to metallic-roughness using the conversion mechanism described in the [appendix](#appendix). Such a conversion during load can be slow and potentially lossy so the runtime could also choose to ignore the material parameters and render the asset using 50% gray emissive color as described in the [base specification](/specification/2.0/README.md#appendix-a).
+
+If an asset includes the pbrSpecularGlossiness extension using the `extensionsRequired` property and does not specify any metallic-roughness properties then only a runtime that supports the pbrSpecularGlossines extension can render it. With this approach you trade off portability of the asset to ensure quality.  
+
+The following table describes the expected rendering behavior based on the material definitions included in the asset:
+
```

Comment:
> the asset isn't following the core spec anymore

Right. But a runtime should support core spec anyway. So in what case does last column apply? Are we basically allowing to implement only SpecGloss and always convert MR?